dev/breeze/src/airflow_breeze/commands/kubernetes_commands.py

```diff
@@ -166,6 +165,13 @@ def kubernetes_group():
     is_flag=True,
     envvar="UPGRADE",
 )
+option_use_docker = click.option(
+    "--use-docker",
+    help="Use Docker to start k8s executor (otherwise k9s from PATH is used and only"
+    " run with docker if not found on PATH).",
+    is_flag=True,
+    envvar="USE_DOCKER",
+)
```

Comment:
Maybe `--use-local-k9s`? By the same way it works for hatch build, use docker by default if not specified local?

Reply:
Well - I think this is better (k9s has no AMD image so local is better if it is installed). The idea about that command was to make it works even if you never installed k9s, but if you did and it is on the PATH, then I think it's better to use it (unless you force docker)

With hatch it's different case and goal actually.

There the fact that we are using docker is a matter of security, when you prepare releases, generally we want to do it in docker by default - even if it is slower - because this shields you from (and same in CI) from unintentionally running a code that someone submitted in PR from fork on your environment - see https://airflow.apache.org/docs/apache-airflow/stable/configurations-ref.html#reload-on-plugin-change - that's why the `breeze release-management prepare-airflow-package` uses docker by default (and only has --use-local as a switch when you know what you are doing.

Reply:
(wrong link - correected it)

Reply:
I've also thinking about running native k9s on docker on ARM. The only way which I found it is build [original Dockerfile](https://github.com/derailed/k9s/blob/master/Dockerfile) and named it `derailed/k9s`. 
But what if we add  option to specify image (default to `derailed/k9s`)? Or it is overengineering and better use local in this case?

```console
❯ docker build . -t derailed/k9s
[+] Building 288.0s (15/15) FINISHED                                                                                                                                docker:desktop-linux
 => [internal] load build definition from Dockerfile                                                                                                                                0.0s
 => => transferring dockerfile: 1.00kB                                                                                                                                              0.0s
 => [internal] load metadata for docker.io/library/alpine:3.19.1                                                                                                                    1.5s
 => [internal] load metadata for docker.io/library/golang:1.21.5-alpine3.17                                                                                                         1.6s
 => [internal] load .dockerignore                                                                                                                                                   0.0s
 => => transferring context: 161B                                                                                                                                                   0.0s
 => [build 1/6] FROM docker.io/library/golang:1.21.5-alpine3.17@sha256:92cb87af996ec6befc85f0aec27e12ead2fab396695fa8a7abff79e021e58195                                             0.0s
 => [internal] load build context                                                                                                                                                   0.1s
 => => transferring context: 1.90MB                                                                                                                                                 0.1s
 => [stage-1 1/3] FROM docker.io/library/alpine:3.19.1@sha256:c5b1261d6d3e43071626931fc004f70149baeba2c8ec672bd4f27761f8e1ad6b                                                      0.0s
 => => resolve docker.io/library/alpine:3.19.1@sha256:c5b1261d6d3e43071626931fc004f70149baeba2c8ec672bd4f27761f8e1ad6b                                                              0.0s
 => => sha256:c5b1261d6d3e43071626931fc004f70149baeba2c8ec672bd4f27761f8e1ad6b 1.64kB / 1.64kB                                                                                      0.0s
 => => sha256:a0264d60f80df12bc1e6dd98bae6c43debe6667c0ba482711f0d806493467a46 528B / 528B                                                                                          0.0s
 => => sha256:ace17d5d883e9ea5a21138d0608d60aa2376c68f616c55b0b7e73fba6d8556a3 1.49kB / 1.49kB                                                                                      0.0s
 => CACHED [build 2/6] WORKDIR /k9s                                                                                                                                                 0.0s
 => CACHED [build 3/6] COPY go.mod go.sum main.go Makefile ./                                                                                                                       0.0s
 => [build 4/6] COPY internal internal                                                                                                                                              0.2s
 => [build 5/6] COPY cmd cmd                                                                                                                                                        0.0s
 => [build 6/6] RUN apk --no-cache add --update make libx11-dev git gcc libc-dev curl && make build                                                                               257.0s
 => [stage-1 2/3] COPY --from=build /k9s/execs/k9s /bin/k9s                                                                                                                         0.3s 
 => [stage-1 3/3] RUN apk add --update ca-certificates   && apk add --update -t deps curl vim   && TARGET_ARCH=$(arch | sed s/aarch64/arm64/ | sed s/x86_64/amd64/)   && curl -L   27.9s 
 => exporting to image                                                                                                                                                              0.4s 
 => => exporting layers                                                                                                                                                             0.4s 
 => => writing image sha256:b058d99a0b0176c72c983875e2005950be898543e498bbb90103a720028834e8                                                                                        0.0s 
 => => naming to docker.io/derailed/k9s                                                                                                                                             0.0s

❯ docker run -it --rm derailed/k9s info
 ____  __.________       
|    |/ _/   __   \______
|      < \____    /  ___/
|    |  \   /    /\___ \ 
|____|__ \ /____//____  >
        \/            \/ 

Version:           v0.31.8
Config:            /root/.config/k9s/config.yaml
Custom Views:      /root/.config/k9s/views.yaml
Plugins:           /root/.config/k9s/plugins.yaml
Hotkeys:           /root/.config/k9s/hotkeys.yaml
Aliases:           /root/.config/k9s/aliases.yaml
Skins:             /root/.config/k9s/skins
Context Configs:   /root/.local/share/k9s/clusters
Logs:              /root/.local/state/k9s/k9s.log
Benchmarks:        /root/.local/state/k9s/benchmarks
ScreenDumps:       /root/.local/state/k9s/screen-dumps

❯ docker inspect derailed/k9s | grep Architecture
        "Architecture": "arm64",
```

Reply:
> Or it is overengineering and better use local in this case?

Yeah. I think better use local, especailly when it's as easy as `brew install k9s` on Mac